var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -143,12 +144,24 @@ class QtBase(QtPackage):
             depends_on("libxrender")
 
     with when("+network"):
-        depends_on("libproxy")
+        if not bool(MACOS_VERSION):
```

Comment:
In python more simply:
```suggestion
        if not MACOS_VERSION:
```